src/shared/src/types/reimbursement-requests-types.ts

```diff
@@ -80,7 +80,7 @@ export interface ReimbursementReceiptCreateArgs {
 }
 
 export interface ReimbursementReceiptUploadArgs extends ReimbursementReceiptCreateArgs {
-  file: File;
+  file?: File;
```

Comment:
a type error in the EditRRDefault values file

Reply:
i dont like making this optional

Reply:
I can make an alternate type for default values if you want then. The weirdness is because we only need that file data the first time a receipt is added, since we don't reupload it everytime a RR is edited

Reply:
which now that I think ab it does make it optional

Reply:
cause sometimes we include the file data and sometimes we don't

Reply:
thats true...

Reply:
i tried hard to use interface segregation rather than optionals with these files but if it makes sense to make it optional then we should do that